frontend/src/components/Events/EventsList.js

```diff
@@ -33,6 +33,11 @@ const EventsList = () => {
   const isAdmin = keycloak.hasRealmRole('admin');
   const eventsResponse = useTimeKeeperAPI('/api/events');
 
+  // Sort events by startDateTime order (DESC)
+  if(eventsResponse.data){
+    eventsResponse.data.sort((a,b) => a.startDateTime > b.startDateTime ? -1 : a.startDateTime < b.startDateTime ? 1 : 0);
```

Comment:
Does eventsResponse.data is supposed to be mutable ? Because else, I don't know how the events can be sorted.
What is the type of startDateTime ? I think you are supposed to transform the dates (string) to moment then compare them with isAfter. Now, you seems to compare two strings.
You can also use orderBy by lodash.